src/main/java/rsb/plugin/ScriptPanel.java

```diff
@@ -143,6 +178,8 @@ private void assignLayouts() {
 										.addComponent(buttonPause, GroupLayout.PREFERRED_SIZE, 33, GroupLayout.PREFERRED_SIZE))
 								.addGap(28, 28, 28)
 								.addComponent(buttonStop, GroupLayout.PREFERRED_SIZE, 36, GroupLayout.PREFERRED_SIZE)
+								.addGap(28, 28, 28)
```

Comment:
I'm not sure when this "setVerticalGroup" gets called, so no guarantee how it looks here